src/components/BestSeller.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useRef, useState } from "react";
+import { Virtual, Navigation, Pagination } from "swiper/modules";
+import { Swiper, SwiperSlide } from "swiper/react";
+import BestSellerCard from "@/components/BestSellerCard";
+import p12 from "@/assets/p12.png";
+import p15 from "@/assets/p15.png";
+
+
+import "swiper/css";
+import "swiper/css/pagination";
+import "swiper/css/navigation";
+
+export const BestSellerArr = [
```

Comment:
use camelCase not Pascal case for variable

Reply:
Changed all the variable names to PascalCase to camelCase.